climpyrical/gridding.py

```diff
@@ -432,26 +513,32 @@ def check_find_nearest_value_inputs(x, y, x_i, y_i, field, mask):
     ):
         raise ValueError(
             "Both index array must contain integers. Received \
-            {} and {}".format(x_i.dtype, y_i.dtype)
+            {} and {}".format(
+                x_i.dtype, y_i.dtype
+            )
         )
     if (x_i.max() > x.size) or (y_i.max() > y.size):
         raise ValueError(
             "Indices in index arrays are larger than coordinate array size. \
-            Received x, y {},{} with sizes {}, {}."
-            .format(x_i.max(), y_i.max(), x.size, y.size)
+            Received x, y {},{} with sizes {}, {}.".format(
+                x_i.max(), y_i.max(), x.size, y.size
+            )
         )
     # field same shape as xiyi
     if field.shape != (y.size, x.size):
         raise ValueError(
             "Field provided is not consistent with coordinates provided. \
-            Recevied field shape {}, expected shape ({},{})"
-            .format(field.shape, y.size, x.size)
+            Recevied field shape {}, expected shape ({},{})".format(
+                field.shape, y.size, x.size
+            )
         )
     # mask same shape as field
     if field.shape != mask.shape:
         raise ValueError(
             "Field and mask are not the same shape. Received field shape \
-            {} and mask shape {}.".format(field.shape, mask.shape)
+            {} and mask shape {}.".format(
+                field.shape, mask.shape
+            )
         )
```

Comment:
Wow, you are totally hardcore checking inputs.

Reply:
:nerd_face: